src/Dialect/ONNX/AdditionalONNXOps.td

```diff
@@ -21,3 +24,77 @@ def ONNXCustomOp:ONNX_Op<"Custom",
     }
   }];
 }
+
+def ONNXCallOp : ONNX_Op<"ONNX_Call",
```

Comment:
This seems to be code from #987 Why is it here?

Reply:
That's fine. This is an issue with the PR not recognizing that Kevin's PR already got merged. I was going to raise the issue during our next meeting. It didn't use to do this. I remember we were talking about changing the way things get merged to keep the commit log clean. I don't know if that was done and was causing the trouble.

Basically, now whenever I rebase my PR after new commits got merged into master, it doesn't recognize that and shows those new commits onto which I just rebased as being my commits. Very annoying indeed.